styles/globals.scss

```diff
@@ -1,9 +1,14 @@
 @import "./abstracts/variables";
-@import "./components/navigation";
 @import "./base/base";
 @import "./base/typography";
+@import "./components/navigation";
+@import "./components/stamp";
+@import "./components/socialMedia";
+@import "./components/sitemap";
 @import "./components/title";
 @import "./components/subtitle";
+@import "./layout/container";
+@import "./layout/footer";
 
 body {
```

Comment:
Remember that base styles for tags should be in the base.scss not in the globals, globals should only have the imports